### PR TITLE
Towery - pull request assignment one

### PR DESCRIPTION
**Pull request by jaamp**
Towery - pull request assignment one - four HTML

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            <label>
+                Email Address:
+                <input type="email" name="email">
+            </label>
+            <label>
+                Street Address:
+                <input type="text" name="street">
+            </label>
+            <label>
+                City:
```
Comment (ClovisI) [inline contact.html]: You may want to put the City, State, and Zip in a container to keep them on the same line.

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles/main-style.css">
+        <link rel="stylesheet" media="screen and (max-width: 500px)" href="styles/width-small.css">
+        <link rel="stylesheet" media="screen and (min-width: 501px) and (max-width: 850px)" href="styles/width-medium.css">
+        <link rel="stylesheet" media="screen and (min-width: 851px)" href="styles/width-large.css">
+        <title>Dog Blog</title>
+    </head>
+    <body>
+            <header class="header-container">
```
Comment (jimmynono) [inline blog.html]: Watch your indenting.

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    <div class="header-item-1"><img src="images/cart-icon.svg" alt="cart icon">
```
Comment (jimmynono) [inline blog.html]: When you wrap things in divs make sure to do it like this

```
<div>
  <h1>Hi there</h1>
</div>

```
Also you probably don't need all of these divs in here.

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                <li><a href="blog.html">Dog Blog</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+        <section class="top-section">
+            <h1 class="home-h">Adoptadog Blog</h1>
+            <hr/>
+            <h2 class="home-h">Traveling With Your Dog</h2>
+            <section class="flex-large">
+                <img class="home-image"  src="images/blog-1.jpg" alt="dog blog one">
```
Comment (jimmynono) [inline blog.html]: make sure your alt tags are actually descriptive

File: blog.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+                <img class="home-image"  src="images/blog-3.jpg" alt="dog blog three">
+              
+                <p class="home-p">Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.</p>
+                <p class="home-p">Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+            </section>
+        </section>
+        <hr/>
+        <hr/>
+        <footer>
+                <img class="header-item-3" src="images/adoptadog-logo.svg" alt="dog logo">
```
Comment (jimmynono) [inline blog.html]: indenting

File: dogs.html [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+        <ul class="nav-ul">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li><a href="blog.html">Dog Blog</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+    <section>
+        <h1>Here are all of our Dogs!</h1>
+        <div class="card-parent">
```
Comment (jimmynono) [inline dogs.html]: maybe you can find a more semantic name for this?

This organization and layout is smart, it'll make your layout work great!

File: dogs.html [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+                <div class="card">
+                    <img class="card-img" src="images/murphy-card.jpg" alt="murphy">
+                    <div class="card-container">
+                        <h3>Murphy</h3>
+                        <p>Cost to Adopt: $123.45</p>
+                        <p> Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae
+                        </p>
+                        <input type="button" value="Adopt">
```
Comment (jimmynono) [inline dogs.html]: you can just use the button element

File: dogs.html [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+                        Corrum volorit iandae
+                        nimaxim cum restia volor
+                        reicid ut et etur sunt arum
+                        rendae pla endis re ea erum,
+                        qui doluptae
+                    </p>
+                    <input type="button" value="Adopt">
+                </div>
+            </div>
+            <div class="card">
```
Comment (jimmynono) [inline dogs.html]: keep an eye on your indenting. All of these cards should all be aligned

File: styles/main-style.css [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+  .card-container {
+    padding: .5em 1em;
+    text-align: center;
+
+  }
+.card-img {
+    border-radius: 1em 1em 0 0;
+    width: 100%;
```
Comment (jimmynono) [inline styles/main-style.css:73]: YES!

File: styles/width-large.css [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  
+  .header-item-2 {
+    float: right;
+  }
+  
+.header-item-3 {
+  width: 30%;
+}
+
+  .card {
```
Comment (jimmynono) [inline styles/width-large.css:26]: keep an eye on indenting

File: .vscode/launch.json [hunk 11]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (jimmynono) [inline .vscode/launch.json:1]: This file isn't needed, and you should pull it from your repo

File: blog.html [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+      <li><a href="index.html">Home</a></li>
+      <li><a href="dogs.html">Dogs</a></li>
+      <li><a href="blog.html">Dog Blog</a></li>
+      <li><a href="contact.html">Contact</a></li>
+    </ul>
+  </nav>
+  <section class="top-section">
+    <h1 class="home-h">Adoptadog Blog</h1>
+    <hr />
+    <section id="blog-content-id" class="top-section">
```
Comment (jimmynono) [inline blog.html:33]: You'll want to have this whole section empty, and generate the blog posts from JS. With jQuery it's not quite working, but regardless, this should be empty

Comment (jaamp) [reply blog.html:33]: I emptied the section. the form is created by jQuery, and by completing an entry renders a post, also by jQuery, I also changed the jQuery to https link

File: README.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+2. Firefox – no significant differences from above noted.
+
+3. Chrome - no significant differences from above noted.
+
+4. Edge - no significant differences from above noted.
+
+5. IE11 – There were significant issues with rendering content, at the difference media query sizes this was mainly with images on the home page and blog page.  Some of the JavaScript / jQuery did not work with selecting and calculating cart total. Interestingly, the CSS transitions did work well.  Other, issues noted with the other browsers were apparent with IE11 as well.
```
Comment (jimmynono) [inline README.md:28]: IE11 is the devil

**Pull request closed without merging**